exercise/exercise_models.py

```diff
@@ -680,35 +682,38 @@ def load(self, request, students, url_name="exercise"):
             'url': url,
             'parameters': lti.sign_post_parameters(url),
             'title': self.resource_link_title,
+            'exercise': self,
+            'unique_id': lti.get_param('resource_link_id'),
```

Comment:
I would prefer `lti.parameters['resource_link_id']`  as `get_param` indicates it probably returns None if key is missing.